### DISCUSSION TITLE
Advanced Transfer Audio ideas

### DISCUSSION THREAD

**Discussion by manatactical** [apohl79/audiogridder#939] [Ideas]
Hi,

Having the option to pause audio transfer while the DAW is not playing/recording is a great idea! It is saving quite a lot of CPU power. However, with the sandbox option enabled, this param stays global.
It would be wonderful to have the audio transfer being active only while it is receiving MIDI on the server-side, and by processes. Doing so would reduce CPU utilization on silent parts and save bandwidth. Also, it would not require the DAW to be playing to try out an instrument.

Is this doable?

Thanks for reading. 
Loving this app so far!

**Comment by apohl79** [comment]
I think i understand where you are coming from. Where would MIDI data come from in an FX chain? Or is this for an instruments scenario, where FX plugins would follow an instrument?

**Reply by manatactical** [reply]
I forgot to mention it's about the instrument version of the plugin yes.
It doesn't concern the FX. I don't use AG as audio insert, so I tend to forget about it sorry :D

**Reply by apohl79** [reply]
understood, will create a feature request.
